src/assets/locales/de/registration.json

```diff
@@ -4,12 +4,13 @@
     "companyHeading": "Firmendetails",
     "companyMessage": "Unten finden Sie die Registrierungsdaten Ihres Unternehmens, die Ihnen von Ihrem Onboarding-Anbieter mitgeteilt wurden. Bitte überprüfen Sie die Details und fahren Sie mit der Auswahl der Unternehmensrolle unten fort, bevor Sie Ihre Registrierung absenden.",
     "companyName": "Name der Firma",
-    "bmw": "BMW AG",
```

Comment:
is this deleted on purpose?

Reply:
Yes, it was removed from the form but left in the transition files unused.